actionmailer/lib/action_mailer/test_helper.rb

```diff
@@ -123,7 +123,7 @@ def assert_enqueued_emails(number, &block)
     #       ContactMailer.with(email: 'user@example.com').welcome.deliver_later
     #     end
     #   end
-    def assert_enqueued_email_with(mailer, method, args: nil, queue: "mailers", &block)
+    def assert_enqueued_email_with(mailer, method, args: nil, queue: "default", &block)
```

Comment:
I think the right fix is to change this to use the computed value based on the config.

Reply:
Updated